client/app/bundles/course/assessment/submission/submissions.js

```diff
@@ -0,0 +1,186 @@
+import getUrlParameter from 'lib/helpers/url_helpers';
+
+/* eslint wrap-iife: ["error", "inside"] */
+/* eslint no-use-before-define: ["error", { "functions": false }] */
+/* global $:false, jQuery:false */
+
+const DOCUMENT_SELECTOR = '.course-assessment-submission-submissions.edit ';
+const SINGLE_QUESTION_ASSESSMENT_SELECTOR = `${DOCUMENT_SELECTOR}.single-question `;
+const MULTI_QUESTION_ASSESSMENT_SELECTOR = `${DOCUMENT_SELECTOR}.multi-question `;
+const SUMMARY_GRADE_SELECTOR = '.submission-grades-summary-grade';
+const GRADE_INPUT_SELECTOR = 'input.grade';
+const POINTS_AWARDED_SELECTOR = 'input.submission-points-awarded';
+const MAXIMUM_GRADE_SELECTOR = '.submission-statistics-maximum-grade';
+const TOTAL_GRADE_SELECTOR = '.submission-statistics-total-grade';
+
+  /**
+   * Update the initial EXP points after page load.
+   */
+function updateInitialPoints() {
+  if ($(DOCUMENT_SELECTOR).length !== 0) {
+    const totalGrade = Number($(TOTAL_GRADE_SELECTOR).text());
+    if (isNaN(totalGrade)) { return; }
+
+    const assignedExp = $(POINTS_AWARDED_SELECTOR).val();
+    if (!assignedExp) {
+      updateExperiencePointsAwarded(totalGrade);
+    }
+  }
+}
+  /**
```

Comment:
@naomilwx the indentation of docs are not consistent with methods in both files.